client/dist/style.css

```diff
@@ -1,20 +1,26 @@
 .main {
 	display: inline-block;
-	border: 1px solid black;
+	border: 1px solid grey;
```

Comment:
If you go to developer-element in Nordstrom and click an element you want to match, you can find an exact color they used!